nat/service_pfctl.go

```diff
@@ -38,17 +38,9 @@ func (service *servicePFCtl) Add(rule RuleForwarding) {
 }
 
 func (service *servicePFCtl) Start() error {
-	err := service.ipForward.Enable()
-	if err != nil {
-		return err
-	}
-
+	service.ipForward.Enable()
```

Comment:
Warning logging whould be exactly here in high logic.
So that everybody understands why we swallow errors

Reply:
We have logging in the `Enable`, do you want to duplicate it or move it to `Start` only?

Reply:
Added extra log line.